content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -16,21 +16,22 @@ weight: 10
 
 {{< feature-state for_k8s_version="v1.17" state="beta" >}}
 
-_Endpoint Slices_ provide a simple way to track network endpoints within a
+_EndpointSlices_ provide a simple way to track network endpoints within a
 Kubernetes cluster. They offer a more scalable and extensible alternative to
 Endpoints.
```

Comment:
In a way, it's kind of handy that there isn't already a concept page for Endpoint.